plugins/modules/ec2_ami.py

```diff
@@ -474,6 +473,12 @@ def create_image(module, connection):
                 device = rename_item_if_exists(device, 'volume_size', 'VolumeSize', 'Ebs', attribute_type=int)
                 device = rename_item_if_exists(device, 'iops', 'Iops', 'Ebs')
                 device = rename_item_if_exists(device, 'encrypted', 'Encrypted', 'Ebs')
+
+                if 'NoDevice' in device:
+                    if device['NoDevice'] is True:
```

Comment:
Could you add a brief comment here explaining why this is being done?  Maybe even just a link to the boto or API reference, since this option is a little odd.